examples/04-grav/Chile_GRAV_4_Miller/LdM_grav_obs.grv

```diff
@@ -0,0 +1,192 @@
+191				
```

Comment:
Would you mind removing these (all of the Chile_GRAV_4_Miller files + the tarball)? The script includes the instructions for downloading them, so we do not need to include them in the repo. We should also update the gitignore to not include these files